bigbluebutton-html5/imports/ui/components/chat/message-list/message-list-item/component.jsx

```diff
@@ -169,7 +169,7 @@ export default class MessageListItem extends Component {
             <div className={styles.messages}>
               {messages.map(message => (
                 <Message
-                  className={styles.message}
+                  className={(message.text.includes('href') ? styles.hyperlink : styles.message)}
```

Comment:
We went through the same in https://github.com/bigbluebutton/bigbluebutton/pull/6809 - If I sent a message "hi no hrefs here", it would be marked as a link. Could you narrow down the filter?